project/make/cross

```diff
@@ -28,6 +30,21 @@ for platform in $DOCKER_CROSSPLATFORMS; do
 			export LDFLAGS_STATIC_DOCKER="" # we just need a simple client for these platforms
 			export BUILDFLAGS=( "${ORIG_BUILDFLAGS[@]/ daemon/}" ) # remove the "daemon" build tag from platforms that aren't supported
 		fi
+		if [ $GOARCH = "arm" ] && [ $GOOS = "linux" ]
+		then
```

Comment:
@WarheadsSE If you mean for arm-linux-gnueabi-gcc they are the defaults from https://wiki.ubuntu.com/ToolChain/CompilerFlags

CGO itself has no default CXXFLAGS.